manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/UsingTest.scala

```diff
@@ -19,61 +19,140 @@
+    }
+    section("Introduction") {
+      p("""When executing a query, Neo4j needs to decide where in the query graph to start matching.
+          |This is done by looking at the `MATCH` clause and the `WHERE` conditions and using that information to find useful indexes, or other starting points.""")
+      p("""The selected index might not be the best choice though -- sometimes multiple indexes could be used, and Neo4j
+          |has picked the wrong one (from a performance point of view), and sometimes (albeit rarely) it is better to not use an index at all.""")
+      p("""You can force Neo4j to use a specific starting point through the `USING` clause. This is called giving a planner hint.
+          |There are three types of planner hints: index hints, scan hints, and join hints.""")
+      note {
+        p("You cannot use planner hints if your query has a `START` clause.")
+      }
+      p("The following graph is used for the examples below:")
+      graphViz()
+      query(s"$matchString RETURN 1 AS $columnName", assertIntegersReturned(1)) {
+        p("""The following query will be used in some of the examples on this page. It has been intentionally constructed in
+            |such a way that the statistical information will be inaccurate for the particular subgraph that this query
+            |matches. For this reason, it can be improved by supplying planner hints.""")
+        profileExecutionPlan()
+      }
+    }
+    section("Index hints") {
+      p("""Index hints are used to specify which index, if any, the planner should use as a starting point.
+          |This can be beneficial in cases where the index statistics are not accurate for the specific values that
+          |the query at hand is known to use, which would result in the planner picking a non-optimal index.
+          |To supply an index hint, use `USING INDEX variable:Label(property)` after the applicable `MATCH` clause.""")
+      p("""It is possible to supply several index hints, but keep in mind that several starting points
+          |will require the use of a potentially expensive join later in the query plan.""")
+      section("Query using an index hint") {
+        p("""The query above will not naturally pick an index to solve the plan.
+            |This is because the graph is very small, and label scans are faster for small databases.
+            |In general, however, query performance is ranked by the dbhit metric, and we'll see that using an index is
+            |slightly better for this query.""")
+        query(s"$matchString USING INDEX liskov:Scientist(name) RETURN liskov.born AS $columnName",
+              assertIntegersReturned(1939)) {
+          p("Returns the year Barbara Liskov was born.")
+          profileExecutionPlan()
+        }
+      }
+      section("Query using multiple index hints") {
+        p("""Supplying one index hint changed the starting point of the query, but the plan is still 'straight', i.e. it has
```

Comment:
I rephrased to `but the plan is still linear, meaning it only has one starting point`.